telco-examples/mgmt-cluster/eib/custom/files/clusterctl.yaml

```diff
@@ -0,0 +1,7 @@
+providers:
+  - name: "rke2"
+    url: "https://github.com/rancher-sandbox/cluster-api-provider-rke2/releases/v0.2.2/bootstrap-components.yaml"
```

Comment:
v0.2.4 is now out. Do we want to use it?

Reply:
I didn't test it yet, I prefer to keep this version until we can have time to test it